src/config/fix_inversion.json

```diff
@@ -731,6 +731,10 @@
                 ".numberedequation, .displayformula"
             ]
         },
+        {
+            "url": "xda-developers.com",
+            "invert": "div.thumb_meta, i, div#header.header-logged-out.portal-header,div#header.header-logged-in.portal-header,div#header.header-logged-out,div#header.header-logged-in, span"
```

Comment:
1. There is a rule called "CSS Specificity". To describe it in simple words: `#id` gives you 100 points, `.class` gives 10 points, `div` gives 1 point. `div#header` will give you 101 points. So, you can just leave two selectors `.thumb-meta, #header` in this case.
2. `span` selector is very generic, the rule will be applied to elements you don't expect. Try to find a more specific one.
3. If the "invert" value is too long, you can split it into multiple lines like
```json
{
    "invert": [
        "selector1",
        "selector2"
    ]
}